### ISSUE TITLE
[docs] IconGenie 2.3.3 expects string value for splashscreenIconRatio in profile json

### ISSUE THREAD

**Issue by jbwl** [quasarframework/quasar#7908]
I think it is not obvious that the numeric value 0 for splashscreenIconRatio does not work when using a profile, but will be interpreted as not set (40% ratio).

If 0 is wanted, it needs to be a string value in the profile file like this:

```
{
  "params": {
    "include": "all",
    "icon": "icongenie/app-icon.png",
    "splashscreenIconRatio": "0",
    "background": "icongenie/app-splashscreen.png",
    "quality": 12,
    "themeColor": "FFFFFF",
    "skipTrim": true,
    "padding": [
      0,
      0
    ]
  },
  "assets": []
}
```

**Comment by bazuka5801** [comment]
@jbwl Hi :)
You write profile manual?
How I remember, `icongenie` command have mode `generate`, to generate profile, and we only create profile with command or generate icons without profile.

**Comment by bazuka5801** [comment]
But I think it must be fixed on docs, cuz it's confuse.
I intersect with same problem in another project [cordova-plugin-firestore](https://github.com/ReallySmallSoftware/cordova-plugin-firestore)
where defined config empty object must be undefined instead of empty object and I spent few hours to resolve 😂.

**Comment by jbwl** [comment]
Yes, I wrote the profile manually, it's easy to make a mistake if numeric values are actually expected to be chars. Thank you for looking into it. Just a very small thing ;-)

**Comment by rstoenescu** [comment]
Fix is available in "@quasar/icongenie" v2.4.1
